### PR TITLE
adding binary to int conversion

### PR DESCRIPTION
**Pull request by ldrazic**
**Describe the PR**
adding binary to int conversion

**Relation issue**
https://github.com/swaggo/swag/issues/1466

**Additional context**
All the context is in the issue

### CODE REVIEW

File: package.go [hunk 1]
```diff
@@ -94,6 +94,17 @@ func (pkg *PackageDefinitions) evaluateConstValue(file *ast.File, iota int, expr
...
 		switch valueExpr.Kind {
 		case token.INT:
+			// binary
+			if len(valueExpr.Value) > 2 && valueExpr.Value[0] == '0' && (valueExpr.Value[1] == 'b' || valueExpr.Value[1] == 'B') {
+				if x, err := strconv.ParseInt(valueExpr.Value[2:], 2, 64); err == nil {
+					return int(x), nil
+				} else if x, err := strconv.ParseUint(valueExpr.Value[2:], 2, 64); err == nil {
+					return x, nil
+				} else {
+					panic(err)
```
Comment (ubogdan) [inline package.go:104]: You shouldn't panic here. 
You should `return 0, nil`, and print the actual error.

**Review by ubogdan — CHANGES_REQUESTED** [review]

**Comment by ubogdan** [comment]
Please also add unit tests for this change.

File: package.go (hunk 1 above)
Comment (ldrazic) [reply package.go:104]: I thought the same but looking at hexadecimal and octects conversions that are below my code, I tried to keep the same standard

Comment (ubogdan) [reply package.go:104]: Can you please fix the other "standard" :)

**Comment by ubogdan** [comment]
@ldrazic, Any updates?

**Comment by ldrazic** [comment]
@ubogdan I got stuck on the unit tests and left it there

**Comment by sdghchj** [comment]
A same PR as #1593.

**Pull request closed without merging**